javascript/README.md

```diff
@@ -1,52 +1,60 @@
+2. [A quick review](introduction/a_quick_review.md)
+### Organizing your JavaScript Code
+1. [Organizing your JavaScript Code Introduction](organizing_your_javascript_code/organizing_your_javascript_code_introduction.md)
+2. [Objects and Object Constructors](organizing_your_javascript_code/objects_and_object_constructors.md)
+3. [**Project: Library**](organizing_your_javascript_code/project_library.md)
+4. [Factory Functions and the Module Pattern](organizing_your_javascript_code/factory_functions_and_module_pattern.md)
+5. [**Project: Tic Tac Toe**](organizing_your_javascript_code/project_tic_tac_toe.md)
+6. [Classes](organizing_your_javascript_code/classes.md)
+7. [ES6 Modules](organizing_your_javascript_code/es6_modules.md)
+8. [Webpack](organizing_your_javascript_code/webpack.md)
+9. [**Project: Restaurant Page**](organizing_your_javascript_code/project_restaurant_page.md)
+10. [OOP Principles](organizing_your_javascript_code/oop_principles.md)
+11. [**Project: Todo List**](organizing_your_javascript_code/project_todo_list.md)
+### JavaScript in the Real World
+1. [Linting](javascript_in_the_real_world/linting.md)
+2. [Dynamic User Interface Interactions](javascript_in_the_real_world/dynamic_user_interface_interactions.md)
+3. [Form Validation with JavaScript](javascript_in_the_real_world/form_validation_with_javascript.md)
+4. [What is ES6?](javascript_in_the_real_world/what_is_es6.md)
+### Asynchronous JavaScript and APIs
+1. [JSON](asynchronous_javascript_and_apis/json.md)
+2. [Asynchronous Code](asynchronous_javascript_and_apis/asynchronous_code.md)
+3. [Working with APIs](asynchronous_javascript_and_apis/working_with_apis.md)
+4. [Async and Await](asynchronous_javascript_and_apis/async_and_await.md)
+5. [**Project: Weather App**](asynchronous_javascript_and_apis/project_weather_app.md)
+### Testing JavaScript
+1. [Testing Basics](testing_javascript/testing_basics.md)
+2. [**Project: Testing Practice**](testing_javascript/project_testing_practice.md)
+3. [More Testing](testing_javascript/more_testing.md)
+4. [**Project: Battleship**](testing_javascript/project_battleship.md)
+### React JS
+1. [React Introduction](react_js/react_introduction.md)
+2. [State and Props](react_js/state_and_props.md)
+3. [Handle Inputs and Render Lists](react_js/handle_inputs_and_render_lists.md)
+4. [**Project: CV Application**](react_js/project_cv_application.md)
+5. [Lifecycle Methods](react_js/lifecycle_methods.md)
+6. [Hooks](react_js/hooks.md)
+7. [**Project: Memory Card**](react_js/project_memory_card.md)
+8. [Router](react_js/router.md)
+9. [React Testing Part 1](react_js/react_testing_part_one.md)
+10. [React Testing Part 2](react_js/react_testing_part_two.md)
+11. [**Project: Shopping Cart**](react_js/project_shopping_cart.md)
+12. [Advanced Concepts](react_js/advanced_concepts.md)
+### JavaScript and the Backend
+1. [Using BaaS For Your Back End](javascript_and_the_backend/using_baas_for_your_backend.md)
```

Comment:
The BaaS lesson is exclusive to the Full Stack JavaScript path, whereas we have a "Using Ruby on Rails For Your Backend" lesson exclusive to the Full Stack Ruby on Rails path. We should update the README to reflect this:
```suggestion
1. - [Using BaaS For Your Back End](javascript_and_the_backend/using_baas_for_your_backend.md) (Exclusive to the Full Stack JavaScript path)
   - [Using Ruby on Rails For Your Back End](add_file_link_here) (Exclusive to the Full Stack Ruby on Rails path)
```